packages/docsearch-css/src/modal.css

```diff
@@ -576,10 +585,9 @@ assistive tech users */
   }
 
   .DocSearch-Container {
-    height: 100vh;
```

Comment:
this line is not necessary. 

`vh` feature/unit came after `calc`. In your `calc` function, you use `1vh * 100` as a fallback. This means this line is redundant.